app/views/notifications/index.erb

```diff
@@ -14,23 +14,23 @@
 
   <div class="row">
     <div class="col-xs-8">
-      <% if inbox.has_unconfirmed_team_memberships? %>
+      <% if inbox.has_team_membership_invites? %>
         <ul style="list-style-type: none; margin: 0;">
-          <% inbox.unconfirmed_team_memberships.each do |team_membership| %>
+          <% inbox.team_membership_invites.each do |invite| %>
           <li class="alert-row">
             <span style="padding-right: 15px;" class="fa fa-bell-o alert-icon"></span>
-            <% if team_membership.inviter.present? %>
-              <%= gravatar_tag team_membership.inviter.avatar_url, size: 20 %>
-              <a href="<%= team_membership.inviter.username %>"><%= team_membership.inviter.username %></a> would like you to join the team <%= team_membership.team.name %>.
+            <% if invite.inviter.present? %>
```

Comment:
This will always happen now, but I don't know if we have some records in the DB that are not `confirmed` but have an empty `inviter_id`, so I decided to leave the conditional.